iconrpcserver/dispatcher/validator.py

```diff
@@ -564,8 +564,19 @@ def validate_jsonschema(request: object, schemas: dict = SCHEMA_V3):
     try:
         validator.validate(request)
     except ValidationError as e:
+        print(f"{e.schema_path}, || {e.path}")
+        if e.schema_path[-1] == "additionalProperties":
+            if len(e.path) == 0:
+                message = f"There is an invalid key in 1st depth"
+            else:
+                message = f"There is an invalid key in '{e.path[-1]}'"
+        elif len(e.path) > 0:
+            message = f"'{e.path[-1]}' has an invalid value"
+        else:
+            message = f"Invalid request"
```

Comment:
How about using the standard term: "Invalid params"

Reply:
applied